src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -76,6 +79,10 @@ public function lateCollect()
 
                 if (null !== $template && '' !== $path = $template->getSourceContext()->getPath()) {
                     $this->data['template_paths'][$name] = $path;
+
+                    if (0 === strpos($path, $this->projectDir)) {
```

Comment:
what if  `$this->projectDir` is `null` ?

Reply:
The absolute paths will be shown.

Reply:
not exactly. `strpos(..., null)` will still perform a lookup in the string, to find a `\0` char (it converts `null` to an integer and looks for `ord($int)` in all releases of PHP as it is not a string (and passing something else than a string as needle will trigger a deprecation in PHP 5.3).

See https://3v4l.org/67Okn for the proof (7.3.0alpha1 does not yet have the deprecation, but it will come as the RFC was accepted).

You should add a check for `null` as project dir here in the if, before the call to `strpos`

Reply:
It's good to know, I'll make the changes, thanks for the explanation.

Reply:
Fixed.